core/src/main/java/io/questdb/griffin/model/IntrinsicModel.java

```diff
@@ -82,7 +82,9 @@ public void intersectEmpty() {
 
     public void intersectEquals(Function function) {
         runtimeIntervalBuilder.intersectEquals(function);
-        if (runtimeIntervalBuilder.isEmptySet()) intrinsicValue = FALSE;
+        if (runtimeIntervalBuilder.isEmptySet()) {
+            intrinsicValue = FALSE;
+        }
```

Comment:
Can we improve 72.73% coverage of the changed lines in this class please? Ideally to 100%

Reply:
These LoC are reformatting, so it seems that the class was poorly covered with tests. I'll improve them.

Reply:
Added the tests in cb73c79eac80d68f38e6e52d7c088afb0f862b02